### PR TITLE
Speaker session blog post

### LINKED ISSUE #3373: [BLOG] Session Spotlight - Keynote Address at OpenSearchCon North America 2024
### Describe the blog post

This is the first in a series of posts highlighting the most important sessions from OpenSearchCon North America 2024. This post covers the keynote address on the first day.

### Expected Title

Session Spotlight - Keynote Address at OpenSearchCon North America 2024

### Authors Name

Matt Lutton

### Authors Email

mlutton@steyer.net

### Target Draft Date

_No response_

### Blog Post Category

community

### Target Publication Date

10/11/2024

### Additional Info

_No response_

### PR DESCRIPTION
**Pull request by kaimmej**
### Description
Created blog post for the OpenSearchCon NA 2024 keynote address. This had been written by Matt Lutton. 
I created Matt as a community member. 
I created a new youtube player include that scales a bit differently and has a different set of styles associated with it. I did not change the current player so that previous blog posts were not inadvertantly affected. 
 
### Issues Resolved
#3373 

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2024-10-21-Session-Spotlight-OpenSearchCon-Keynote.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+layout: post
+title:  'Session Spotlight: Keynote Address at OpenSearchCon North America 2024'
+authors:
+ - mlutton
+date: 2024-10-21
+categories:
+ - community-updates
+meta_keywords: OpenSearchCon, OpenSearch Project, Adobe, Linux Foundation, GenAI, Vector Search, OpenSearch Software Foundation
+meta_description: Watch the OpenSearchCon North America 2024 keynote address and hear from featured speakers from OpenSearch, Adobe, and the Linux Foundation as they discuss uses cases along with the project’s history and future.
+excerpt: 'The Opening Keynote for OpenSearchCon North America 2024 was given by Eli Fisher, Senior Manager of Project Management at AWS for OpenSearch, alongside two very special guests: Anupam Chaturvedi, Engineering Manager for Acrobat GenAI Services at Adobe, and Jim Zemlin, Executive Director of the Linux Foundation. '
```
Comment (natebower) [inline _posts/2024-10-21-Session-Spotlight-OpenSearchCon-Keynote.md:11]:
```suggestion
excerpt: 'The opening keynote for OpenSearchCon North America 2024 was given by Eli Fisher, Senior Manager of Project Management at AWS for OpenSearch, alongside two very special guests: Anupam Chaturvedi, Engineering Manager for Acrobat GenAI Services at Adobe, and Jim Zemlin, Executive Director of the Linux Foundation.'
```

File: _posts/2024-10-21-Session-Spotlight-OpenSearchCon-Keynote.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+has_math: false
+has_science_table: false
+youtubeID: WPVGfWoR0B8
+---
+
+Whether or not you were able to join us in San Francisco for OpenSearchCon North America 2024 now is a fantastic time to catch up with recordings of the sessions. In our first Session Spotlight we’re bringing you back to the kickoff of the conference on September 24th, 2024.
```
Comment (natebower) [inline _posts/2024-10-21-Session-Spotlight-OpenSearchCon-Keynote.md:17]:
```suggestion
Whether or not you were able to join us in San Francisco for OpenSearchCon North America 2024, now is a fantastic time to review recordings of the sessions. In our first session spotlight, we're bringing you back to the kickoff of the conference on September 24, 2024. 
```

File: _posts/2024-10-21-Session-Spotlight-OpenSearchCon-Keynote.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+excerpt: 'The Opening Keynote for OpenSearchCon North America 2024 was given by Eli Fisher, Senior Manager of Project Management at AWS for OpenSearch, alongside two very special guests: Anupam Chaturvedi, Engineering Manager for Acrobat GenAI Services at Adobe, and Jim Zemlin, Executive Director of the Linux Foundation. '
+has_math: false
+has_science_table: false
+youtubeID: WPVGfWoR0B8
+---
+
+Whether or not you were able to join us in San Francisco for OpenSearchCon North America 2024 now is a fantastic time to catch up with recordings of the sessions. In our first Session Spotlight we’re bringing you back to the kickoff of the conference on September 24th, 2024. 
+
+The Opening Keynote for OpenSearchCon North America 2024 was given by Eli Fisher, Senior Manager of Project Management at AWS for OpenSearch, alongside two very special guests: Anupam Chaturvedi, Engineering Manager for Acrobat GenAI Services at Adobe, and Jim Zemlin, Executive Director of the Linux Foundation.
```
Comment (natebower) [inline _posts/2024-10-21-Session-Spotlight-OpenSearchCon-Keynote.md:19]:
```suggestion
The opening keynote for OpenSearchCon North America 2024 was given by Eli Fisher, Senior Manager of Project Management at AWS for OpenSearch, alongside two very special guests: Anupam Chaturvedi, Engineering Manager for Acrobat GenAI Services at Adobe, and Jim Zemlin, Executive Director of the Linux Foundation. 
```

File: _posts/2024-10-21-Session-Spotlight-OpenSearchCon-Keynote.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+The Opening Keynote for OpenSearchCon North America 2024 was given by Eli Fisher, Senior Manager of Project Management at AWS for OpenSearch, alongside two very special guests: Anupam Chaturvedi, Engineering Manager for Acrobat GenAI Services at Adobe, and Jim Zemlin, Executive Director of the Linux Foundation. 
+
+{% include embed/youtube.html id=page.youtubeID %}
+
+In this keynote Eli Fisher outlines the history of the OpenSearch Project from inception to this momentous occasion of joining the [Linux Foundation]( ) as the [OpenSearch Software Foundation](http://foundation.opensearch.org) in September 2024. Anupam Chaturvedi also dives in to the exciting use case of GenAI-assisted search in PDF documents at [Adobe](https://www.adobe.com/acrobat/generative-ai-pdf.html), enabled by the powerful vector search capabilities of OpenSearch. And finally, Jim Zemlin makes the case for the bright future of open source software and highlights the strength of partnering with projects like OpenSearch.
```
Comment (natebower) [inline _posts/2024-10-21-Session-Spotlight-OpenSearchCon-Keynote.md:23]:
```suggestion
In this keynote Eli Fisher outlines the history of the OpenSearch Project, from inception to joining the [Linux Foundation]( ) as the [OpenSearch Software Foundation](http://foundation.opensearch.org) in September 2024. Anupam Chaturvedi also dives into the exciting use case of GenAI-assisted search in PDF documents at [Adobe](https://www.adobe.com/acrobat/generative-ai-pdf.html), enabled by the powerful vector search capabilities of OpenSearch. And, finally, Jim Zemlin makes the case for the bright future of open-source software and highlights the strength of partnering with projects like the OpenSearch Project. 
```

File: _posts/2024-10-21-Session-Spotlight-OpenSearchCon-Keynote.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+The Opening Keynote for OpenSearchCon North America 2024 was given by Eli Fisher, Senior Manager of Project Management at AWS for OpenSearch, alongside two very special guests: Anupam Chaturvedi, Engineering Manager for Acrobat GenAI Services at Adobe, and Jim Zemlin, Executive Director of the Linux Foundation. 
+
+{% include embed/youtube.html id=page.youtubeID %}
+
+In this keynote Eli Fisher outlines the history of the OpenSearch Project from inception to this momentous occasion of joining the [Linux Foundation]( ) as the [OpenSearch Software Foundation](http://foundation.opensearch.org) in September 2024. Anupam Chaturvedi also dives in to the exciting use case of GenAI-assisted search in PDF documents at [Adobe](https://www.adobe.com/acrobat/generative-ai-pdf.html), enabled by the powerful vector search capabilities of OpenSearch. And finally, Jim Zemlin makes the case for the bright future of open source software and highlights the strength of partnering with projects like OpenSearch. 
+
```
Comment (natebower) [inline _posts/2024-10-21-Session-Spotlight-OpenSearchCon-Keynote.md:24]: Above: Ensure that LF link is added.

**Review by natebower — COMMENTED** [review]
@kaimmej @pajuric Editorial review complete. Please see my comment and changes and let me know if you have any questions. Thanks!

File: _posts/2024-10-21-Session-Spotlight-OpenSearchCon-Keynote.md (hunk 5 above)
Comment (kaimmej) [reply _posts/2024-10-21-Session-Spotlight-OpenSearchCon-Keynote.md:27]: Hi @natebower , 
Changes have been made!

File: _posts/2024-10-21-Session-Spotlight-OpenSearchCon-Keynote.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+meta_description: Watch the OpenSearchCon North America 2024 keynote address and hear from featured speakers from OpenSearch, Adobe, and the Linux Foundation as they discuss uses cases along with the project’s history and future.
+excerpt: 'The opening keynote for OpenSearchCon North America 2024 was given by Eli Fisher, Senior Manager of Project Management at AWS for OpenSearch, alongside two very special guests: Anupam Chaturvedi, Engineering Manager for Acrobat GenAI Services at Adobe, and Jim Zemlin, Executive Director of the Linux Foundation.'
+featured_blog_post: false 
+featured_image: false 
+has_math: false
+has_science_table: false
+youtubeID: WPVGfWoR0B8
+---
+
+Whether or not you were able to join us in San Francisco for OpenSearchCon North America 2024, now is a fantastic time to review recordings of the sessions. In our first session spotlight we’re bringing you back to the kickoff of the conference on September 24, 2024.
```
Comment (natebower) [inline _posts/2024-10-21-Session-Spotlight-OpenSearchCon-Keynote.md:20]:
```suggestion
Whether or not you were able to join us in San Francisco for OpenSearchCon North America 2024, now is a fantastic time to review recordings of the sessions. In our first session spotlight we're bringing you back to the kickoff of the conference on September 24, 2024. 
```

**Review by natebower — COMMENTED** [review]
@kaimmej Just one curly apostrophe I must have missed the first time. Otherwise, LGTM. Thanks!

Cc: @pajuric

File: _posts/2024-10-21-Session-Spotlight-OpenSearchCon-Keynote.md (hunk 6 above)
Comment (kaimmej) [reply _posts/2024-10-21-Session-Spotlight-OpenSearchCon-Keynote.md]: Fixed!

**Comment by pajuric** [comment]
@nateynateynate @natebower - This blog is ready to push today.

**Review by natebower — APPROVED** [review]
LGTM

**Comment by natebower** [comment]
@nateynateynate @pajuric Approved and merged into `main`.

**Pull request merged**